tutorials/01-Introduction.ipynb

```diff
@@ -0,0 +1,2872 @@
+   "id": "626c8849-9f01-4095-ac0f-dc898a8e348e",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "You can add, subtract, multiply, power, divide like other integer numbers."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 63,
+   "id": "5b00f2e9-27c7-4274-92e1-abfdfa95ba1e",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/latex": [
+       "$\\displaystyle 46$"
+      ],
+      "text/plain": [
+       "46"
+      ]
+     },
+     "execution_count": 63,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "Integer(12) + Integer(34)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 60,
+   "id": "dd91f818-2f68-44b7-a3eb-8219d196d827",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/latex": [
+       "$\\displaystyle 1267650600228229401496703205376$"
+      ],
+      "text/plain": [
+       "1267650600228229401496703205376"
+      ]
+     },
+     "execution_count": 60,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "Integer(2) ** 100"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "075eaa24-1541-40d2-bb5b-89af2962a3c8",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "However, you may notice that they are displayed in $\\LaTeX$ and have custom types than python builtin integers.\n",
+    "\n",
+    "For example, SymPy integers have more richer mathematical functions like `is_prime`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 72,
+   "id": "646f4098-6ff5-46c3-ba74-7781911eed70",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "True"
+      ]
+     },
+     "execution_count": 72,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "Integer(7).is_prime"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "87d87974-21e2-43a7-93fc-37ba10ce991d",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "### Exact Division"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9d848fa0-bf38-4e86-b811-210ebcb34d97",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "You may also notice that the division of integers are exact.\n",
+    "\n",
+    "The division of integers are more mathematical ($\\mathbb{Q}$) than floats."
```

Comment:
We need to go into this in more detail. You can copy text from the [tutorial](https://docs.sympy.org/latest/tutorials/intro-tutorial/gotchas.html#two-final-notes-and) if you like (in general feel free to just copy stuff from the main SymPy tutorial).